markdown/events/2023/bytesize_mag.md

```diff
@@ -21,3 +24,89 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[7:41](https://www.youtube.com/watch?v=IiorfDHeoLo&t=461)
+How does the output of the pipeline look like? Besides all the individual results part of the individual tools, the pipeline generates a clustered heat map showing the MAG abundances across different samples. You can see an example how this looks like and if you would see here for example that certain samples clustered together, for which you know that they are originating from different groups. This might indicate that something has gone wrong. The pipeline also outputs the MAG summary, which I already mentioned. This contains for each bin for each MAG, the abundance information across different samples. The QC matrix from the BUSCO results and the QUAST results, and also taxonomic classifications from the tool GTDP-Tk.
+
+[8:33](https://www.youtube.com/watch?v=IiorfDHeoLo&t=513)
+And with this, I've shown you the rough overview of the pipeline and next I would like to show you some of the impact, different assembly settings can have. For this I simulated some mouse gut data set in the past with the tool CAMISIM, and I generated hybrid data containing Illumina data and Nanopore reads and generated two groups. Each with a time series of four samples. This might be the ideal case where a core assembly might be useful. Now I would like to show you some of the resulting assembly metrics that are commonly used.
+
+[9:14](https://www.youtube.com/watch?v=IiorfDHeoLo&t=554)
+Here you can see for example the total length of the resulting assemblies and then compared for different pipeline ones were different assembly settings were used. The lower two pipeline settings correspond to a sample-wise assembly and using either only short or short and long read, so hybrid data, and the upper two settings correspond to a core assembly. Again with short or short and long reads, and what we can see is that the total length of the resulting assemblies significantly increased both by using the hybrid setting, and by applying the core assembly setting. Similar results we also see when looking at the number of MAGs, so the number of genomes that could be retrieved from this data, and also when looking at the L50 values. This indicates that the actual setting that is used for the assembly within this pipeline can have a relatively huge impact on the results. It's definitely good that the pipeline provides different settings, so that you can really choose the correct setting for input data, and it might also be worth to compare different settings.
```

Comment:
```suggestion
Here you can see, for example, the total length of the resulting assemblies compared for different pipeline runs for which different assembly settings were used. The lower two pipeline settings correspond to a sample-wise assembly and using either only short or short and long reads, so hybrid data, and the upper two settings correspond to a co-assembly, again with short or short and long reads. And what we can see is that the total length of the resulting assemblies significantly increased both by using the hybrid setting, and by applying the co-assembly setting. Similar results we also see when looking at the number of MAGs, so the number of genomes that could be retrieved from this data, and also when looking at the N50 values. This indicates that the actual setting that is used for the assembly within this pipeline can have a relatively huge impact on the results. It's definitely good that the pipeline provides different settings, so that you can really choose the correct setting for your input data, and it might also be worth to compare different settings.
```